### PR TITLE
improved speedup write with F-contiguous arrays

### PR DESCRIPTION
**Pull request by LucaCerina**
Referencing discussion in #92 all Fortran-contiguous arrays are transformed before writing. This vastly improves write speed for this kind of arrays.

### CODE REVIEW

File: pyedflib/edfwriter.py [hunk 1]
```diff
@@ -817,14 +817,14 @@ def writeSamples(self, data_list, digital = False):
             del dataRecord
             dataRecord = np.array([], dtype=np.int32 if digital else None)
             for i in np.arange(len(data_list)):
-                dataRecord = np.append(dataRecord,data_list[i].ravel()[int(ind[i]):int(ind[i]+sampleFrequencies[i])])
+                dataRecord = np.append(dataRecord, data_list[i][int(ind[i]):int(ind[i]+sampleFrequencies[i])])
```
Comment (skjerns) [inline pyedflib/edfwriter.py:820]: now you removed the `ravel`, was this intentional? right now the fix would onyl apply to the highlevel-interface (which is okay I guess)

**Comment by skjerns** [comment]
also in `edfwriter.py` there is the line in 787-791 :) you can just copy&paste the segment from before.

**Comment by LucaCerina** [comment]
You are right, I forgot people may be calling edfwriter directly. As a solution I moved the check at edfwriter level instead of highlevel. Once its done before writing samples, we don't need the other `ravel` calls because 'data_list[i]' is already flattened and C-contiguous

**Comment by skjerns** [comment]
thanks :) looks good now!

**Comment by skjerns** [comment]
oops appareantly the "all checks passed" was a display error and now some builds fail. I'll check it out and fix it if necessary. Should not be a major bug though

**Pull request merged**